aries-site/src/examples/templates/forms/FormContainer.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Tile } from 'aries-core';
+
+export const FormContainer = ({ children, ...rest }) => {
+  return (
+    <Tile
+      background="background-front"
+      border
+      pad={{ horizontal: 'medium', vertical: 'medium' }}
+      {...rest}
+    >
+      {children}
```

Comment:
Unless we need the prop-types definition of children, I think we can just say:
`FormContainer = ({...rest}) ...`
then have 
```
<Tile ... {...rest />
```